packages/sdk-client/src/sdk-middleware-http/http.ts

```diff
@@ -65,6 +65,7 @@ export default function createHttpMiddleware({
   credentialsMode,
   includeResponseHeaders,
   includeOriginalRequest,
+  excludeRequestInErrorResponse = false,
```

Comment:
I would stay consistent with the prefix include. So like `includeRequestInErrorResponse = true`